launcher/MainWindow.cpp

```diff
@@ -397,8 +397,8 @@ class MainWindow::Ui
         actionPatreon = TranslatedAction(MainWindow);
         actionPatreon->setObjectName(QStringLiteral("actionPatreon"));
         actionPatreon->setIcon(LAUNCHER->getThemedIcon("patreon"));
-        actionPatreon.setTextId(QT_TRANSLATE_NOOP("MainWindow", "Support %1"));
-        actionPatreon.setTooltipId(QT_TRANSLATE_NOOP("MainWindow", "Open the %1 Patreon page."));
+        actionPatreon.setTextId(QT_TRANSLATE_NOOP("MainWindow", "Support MultiMC"));
+        actionPatreon.setTooltipId(QT_TRANSLATE_NOOP("MainWindow", "Open the MultiMC Patreon page."));
```

Comment:
We maybe should keep this the same.

Reply:
Why? It's the link to the MultiMC patreon, so it should say MultiMC instead of UltimMC, right?

Reply:
Oh, I think, but what about upstreaming this instead?

Reply:
They won't accept it because what if a fork wants to put their own patreon? Idk or they just deny it

Reply:
In that case, they would add the option for the Patreon, which effectively cancels this out.